systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicUtils.java

```diff
@@ -219,4 +224,32 @@ public static void waitForTopicWithPrefixDeletion(String namespaceName, String t
                 }
             });
     }
+
+    /**
+     * Verifies that {@code absentTopicName} topic remains absent in {@code clusterName} Kafka cluster residing in {@code namespaceName},
+     * for two times {@code topicOperatorReconciliationSeconds} duration (in seconds) of Topic Operator reconciliation time,
+     * by querying the cluster using kafka scripts from {@code queryingPodName} Pod.
+     * @param namespaceName Namespace name
+     * @param queryingPodName  the name of the pod to query Kafka topics from
+     * @param clusterName name of Kafka cluster
+     * @param absentTopicName name of Kafka topic which should not be created
+     * @param topicOperatorReconciliationSeconds interval in seconds for Topic Operator to reconcile
+     * @throws AssertionError in case topic is created
+     */
+    public static void verifyUnchangedTopicAbsence(String namespaceName, String queryingPodName, String clusterName, String absentTopicName, int topicOperatorReconciliationSeconds) {
+
+        long reconciliationDuration = Duration.ofSeconds(topicOperatorReconciliationSeconds).toMillis();
+        long endTime = System.currentTimeMillis() + 2 * reconciliationDuration;
+
+        LOGGER.info("Verify absence of topic '{}' in listed kafka topics for next {} seconds in Namespace/{}", absentTopicName, reconciliationDuration / 1000, namespaceName);
+
+        while (System.currentTimeMillis() < endTime) {
+            assertThat(KafkaCmdClient.listTopicsUsingPodCli(namespaceName, queryingPodName, KafkaResources.plainBootstrapAddress(clusterName)), not(hasItems(absentTopicName)));
+            try {
+                Thread.sleep(Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS);
```

Comment:
I removed 2 local variables,  (`constants.POLL_INTERVAL_FOR_RESOURCE_READINESS) is used directly instead of naming it + `startTime` is removed. 

also I pass here  `topicOperatorReconciliationSeconds` which specifies TO reconciliation time in seconds. (I can obtain this value easily as I am the one setting it in test which call this method).